tests/acceptance/test_mender-artifact.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# Copyright 2016 Mender Software AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from fabric.api import *
+
+import pytest
+import subprocess
+import os
+import re
+import json
+import hashlib
+
+# Make sure common is imported after fabric, because we override some functions.
+from common import *
+
+class TestMenderArtifact:
+    def test_order(self, latest_mender_image):
```

Comment:
Nice. Great that you've added those tests. Maybe we could add somewhere a comment with the link to artifact file description.